docs/ops.html

```diff
@@ -3859,6 +3859,80 @@ <h3>Finalizing the migration</h3>
 
 # Other configs ...</pre>
 
+
+<h3 class="anchor-heading"><a id="tiered_storage" class="anchor-link"></a><a href="#kraft">6.11 Tiered Storage</a></h3>
+
+<h4 class="anchor-heading"><a id="tiered_storage_overview" class="anchor-link"></a><a href="#tiered_storage_overview">Tiered Storage overview</a></h4>
```

Comment:
`Tiered Storage overview` -> `Tiered Storage Overview`

Camel Case is followed for section headings in the docs page.